documentation/modules/ROOT/pages/connectors/jdbc.adoc

```diff
@@ -753,8 +756,7 @@ metadata:
 spec:
   class: io.debezium.connector.jdbc.JdbcSinkConnector  // <2>
   tasksMax: 1  // <3>
-  topics: // <4>
-    - orders
+  topics: orders // <4>
```

Comment:
This needs to be moved into the `config` section where we define things like `connector.class` and `connection.url`.